packages/keybr-keyboard/lib/layout.ts

```diff
@@ -890,9 +890,54 @@ export class Layout implements XEnumItem {
       Geometry.MATRIX,
     ),
   );
+  static readonly TH_TH = new Layout(
+    /* id= */ "th-th",
+    /* xid= */ 0x9a,
+    /* name= */ "Thai Kedmanee",
+    /* family= */ "qwerty",
```

Comment:
Please use `"th-th"` as the layout family value.
Here's why we need layout families.
Many western languages use keyboard layouts which are almost the same as the QWERTY layout.  When, for example, you are switching from English to Spanish, you don't want to start lessons from scratch, because the differences in the layouts are minimal. For this reason we combine all results of layouts from the same family together.
However, all three Thai layouts that you added are very different and unique, so each of them belongs to its own family.